.github/workflows/workflow_inference.yml

```diff
@@ -188,6 +188,14 @@ jobs:
             docker exec "${TARGET}" bash -c "python examples/inference/api_server_openai/query_http_requests.py --model_name ${{ matrix.model }}"
           fi
 
+      - name: Run Agent tool Inference Test with REST API
+        run: |
+          TARGET=${{steps.target.outputs.target}}
+          if [[ ${{ matrix.model }} == "mistral-7b-v0.1" ]]; then
+            docker exec "${TARGET}" bash -c "llm_on_ray-serve --models ${{ matrix.model }}"
+            docker exec "${TARGET}" bash -c "python examples/inference/api_server_openai/query_http_requests_tool.py --model_name ${{ matrix.model }}"
```

Comment:
This is to run the example, can we test and verify the LLM response is a correct Json to call the function?

Reply:
UT verification added, only test on function name, using llama to inference